src/main/scala/ai/privado/entrypoint/ScanProcessor.scala

```diff
@@ -537,15 +559,16 @@ object ScanProcessor extends CommandProcessor {
   private def processCpgWithDefaultProcessor(sourceRepoLocation: String, appCache: AppCache) = {
```

Comment:
As we have removed the function call of this method from above, where else do we use this.

Instead of duplicating code, let's use this method above